### PR TITLE
Add Documentation for Generating Keys and Certificates for TLS Tests

### PR DESCRIPTION
**Pull request by forestmvey**
*Issue #, if available:*

N/A

*Description of changes:

Add documentation covering how to generate TLS keys and certificates for TLS tests. Fixed bug where invalid tests would pass without an invalid certificate in the `integration/tls/cert` directory.

Follow the steps to create certificates and keys in the TLS README.

Execute tests to ensure fix is working with the following commands in the repository root:
```
go clean -testcache
go test -v ./integration/tls
```

Changes Integrated:
- [x] - Validated tls tests pass

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: integration/tls/tls_test.go [hunk 1]
```diff
@@ -40,15 +40,18 @@ const (
 	prometheusConfigPath       = "config/prometheus.yml"
 	prometheusDockerImageName  = "prom/prometheus"
 	tlsRootCAPath              = "cert/RootCA.pem"
+	tlsCertificatePath         = "cert/ServerCertificate.crt"
+	tlsPrivateKeyPath          = "cert/ServerPrivateKey.key"
 	tlsServerCertPath          = "cert"
 	connectorDockerImageName   = "timestream-prometheus-connector-docker"
-	connectorDockerImagePath   = "../../resources/timestream-prometheus-connector-docker-image-1.0.0.tar.gz"
+	connectorDockerImagePath   = "../../resources/timestream-prometheus-connector-docker-image-1.0.3.tar.gz"
```
Comment (trevorbonas) [inline integration/tls/tls_test.go:47]: Can we use `timestream.Version` here?

**Comment by trevorbonas** [comment]
`go test -v integration/tls` didn't work for me, only `go test -v ./integration/tls` worked.

I got the following test failure:
```shell
tls_test.go:118:
                Error Trace:    /Users/trevor.bonas/amazon-timestream-connector-prometheus/integration/tls/tls_test.go:118
                Error:          "0" is not greater than "0"
                Test:           TestHttpsSupport
--- FAIL: TestHttpsSupport (64.63s)
```

File: integration/tls/tls_test.go (hunk 1 above)
Comment (forestmvey) [reply integration/tls/tls_test.go]: Yes, I have revised to now use the imported version file.

**Review by trevorbonas — APPROVED** [review]
Looks good, all tls tests pass on my end.

**Pull request merged**